dag.py

```diff
@@ -0,0 +1,69 @@
+# Lint as: python3
+"""Airflow DAG for generating schemaless and relational files"""
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.utils.dates import days_ago
+
+from schemaless import create_schemaless
+from schemaless import create_uuid_map
+from relational import process_schemaless
+
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,  # TODO
+    'start_date': days_ago(2),
+    'email': ['airflow@example.com'],  # TODO
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=15),
+}
+
+
+dag = DAG(
+    'housing-dashboard-data',
+    default_args=default_args,
+    description=('Consume all available housing data and produce a unified '
+                 'output.'),
+    schedule_interval=timedelta(days=1),
+)
+
+task_create_schemaless = PythonOperator(
+    task_id='create_schemaless',
+    python_callable=create_schemaless.run,
+    op_kwargs={
+        'out_file': '{{ var.value.WORKDIR }}/schemaless.csv',
+        # 'no_download': 'True',
+        'planning_file': '{{ var.value.WORKDIR }}/../data/planning/planning-2020-03-11.csv.xz',  # NOQA
```

Comment:
How come this is explicitly pulled in?

Reply:
This dataset isn't public yet: https://data.sfgov.org/Housing-and-Buildings/SF-Planning-Permitting-Data/kncr-c6jw

Reply:
Shouldn't we also have OEWD here as well then?

Reply:
Yep, fixed